admin-dev/themes/default/template/header.tpl

```diff
@@ -119,97 +119,98 @@
+                <a href="javascript:void(0);" class="ajax-quick-link" data-method="add">
+                  <i class="material-icons">add_circle</i>
+                  {l s='Add current page to QuickAccess' d='Admin.Navigation.Header'}
                 </a>
               </li>
-            </ul>
-          </div>
+            {/if}
+            <li>
+              <a href="{$link->getAdminLink("AdminQuickAccesses")|addslashes}">
+                <i class="material-icons">settings</i>
+                {l s='Manage quick accesses' d='Admin.Navigation.Header'}
+              </a>
+            </li>
+          </ul>
         </div>
-        {$quick_access_current_link_name = " - "|explode:$quick_access_current_link_name}
-        <script>
-          $(function() {
-            $('.ajax-quick-link').on('click', function(e){
-              e.preventDefault();
+      </div>
+      {$quick_access_current_link_name = " - "|explode:$quick_access_current_link_name}
+      <script>
+        $(function() {
+          $('.ajax-quick-link').on('click', function(e){
+            e.preventDefault();
 
-              var method = $(this).data('method');
+            var method = $(this).data('method');
 
-              if(method == 'add')
-                var name = prompt('{l s='Please name this shortcut:' js=1 d='Admin.Navigation.Header'}', '{$quick_access_current_link_name.0|truncate:32}');
+            if(method == 'add')
+              var name = prompt('{l s='Please name this shortcut:' js=1 d='Admin.Navigation.Header'}', '{$quick_access_current_link_name.0|truncate:32}');
 
-              if(method == 'add' && name || method == 'remove')
-              {
-                $.ajax({
-                  type: 'POST',
-                  headers: { "cache-control": "no-cache" },
-                  async: false,
-                  url: "{$link->getAdminLink('AdminQuickAccesses', true, [], ['action' => 'GetUrl', 'rand' => (1|rand:200), 'ajax' => 1])}" + "&method=" + method + ( $(this).data('quicklink-id') ? "&id_quick_access=" + $(this).data('quicklink-id') : ""),
-                  data: {
-                    "url": "{$link->getQuickLink($smarty.server['REQUEST_URI']|escape:'javascript')}",
-                    "name": name,
-                    "icon": "{$quick_access_current_link_icon}"
-                  },
-                  dataType: "json",
-                  success: function(data) {
-                    var quicklink_list ='';
-                    $.each(data, function(index,value){
-                      if (typeof data[index]['name'] !== 'undefined')
-                        quicklink_list += '<li><a href="' + data[index]['link'] + '&token=' + data[index]['token'] + '">' + data[index]['name'] + '</a></li>';
-                    });
+            if(method == 'add' && name || method == 'remove')
+            {
+              $.ajax({
+                type: 'POST',
+                headers: { "cache-control": "no-cache" },
+                async: false,
```

Comment:
Not a good idea^^
   async: false,

Reply:
Not sure I understand why this is not a good idea Besides this code has been changed, only the indentation So it's been here for quite some time I think

Reply:
Synchronous XMLHttpRequest outside of workers is in the process of being removed from the web platform as it has detrimental effects to the end user’s experience. (This is a long process that takes many years.) Developers must not pass false for the async argument when current global object is a Window object. User agents are strongly encouraged to warn about such usage in developer tools and may experiment with throwing an "InvalidAccessError" DOMException when it occurs.

The code being refactored it would be good to remove these depreciated elements, right?